docker/README.md

```diff
@@ -35,15 +35,15 @@ In addition, the following legacy tags are available:
 
 ### [Devito] on GPU
 
-Second, we provide three types of images to run [Devito] on GPUs. These thee images are tagged `devito:nvidia-nvc-*`, `devito:nvidia-clang-*`, and `devito:amd-*`.
+Second, we provide three images to run [Devito] on GPUs, tagged `devito:nvidia-nvc-*`, `devito:nvidia-clang-*`, and `devito:amd-*`.
 
 - `devito:nvidia-nvc-*` is intended to be used on NVidia GPUs. It comes with the configuration to use the `nvc` compiler for `openacc` offloading. This image also comes with CUDA-aware MPI for multi-GPU deployment.
 - `devito:nvidia-clang-*` is intended to be used on NVidia GPUs. It comes with the configuration to use the `clang` compiler for `openmp` offloading. This image also comes with CUDA-aware MPI for multi-GPU deployment.
-- `devito:amd-*` is intended to be used on AMD GPUs. It comes with the configuration to use the `aoompcc` compiler for `openmp` offloading. This image also comes with Rocm-aware MPI for multi-GPU deployment. Additionally, this image can be used on AMD CPUs as well since the Rocm compilers are preinstalled. You will need to modify `DEVITO_PLATFORM` to `amd` at runtime to reflect this architecture.
+- `devito:amd-*` is intended to be used on AMD GPUs. It comes with the configuration to use the `aoompcc` compiler for `openmp` offloading. This image also comes with ROCm-aware MPI for multi-GPU deployment. This image can also be used on AMD CPUs since the ROCm compilers are preinstalled. To reflect this architecture, you will need to modify `DEVITO_PLATFORM` to `amd` at runtime.
```

Comment:
why do we need to change the DEVITO_PLATFORM manually, and not in the dockerfile?

Also, "to reflect this ..." is likely to be bad english